.gitmodules

```diff
@@ -0,0 +1,478 @@
+[submodule "vendor/actionscript"]
```

Comment:
can you use shallow clones (max commits == 1)? No point in cloning the entire histories.

Reply:
Yes, I have updated the docs